dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -194,19 +242,7 @@ public FunctionView Describe()
     /// <inheritdoc/>
     public Task<SKContext> InvokeAsync(SKContext context, CompleteRequestSettings? settings = null)
     {
-        // If the function is invoked manually, the user might have left out the skill collection
-        context.Skills ??= this._skillCollection;
-
-        return this.IsSemantic ?
-            InvokeSemanticAsync(context, settings) :
-            this._function(null, settings, context);
-
-        async Task<SKContext> InvokeSemanticAsync(SKContext context, CompleteRequestSettings? settings)
-        {
-            var resultContext = await this._function(this._aiService?.Value, settings ?? this._aiRequestSettings, context).ConfigureAwait(false);
-            context.Variables.Update(resultContext.Variables);
-            return context;
-        }
+        // ???
```

Comment:
@rracanicci we need to rewrite this from scratch because the underlying implementation has changed /cc @stephentoub